src/main/java/com/example/syncrowbackend/friendrequestroom/dto/FriendRequestPostDto.java

```diff
@@ -0,0 +1,42 @@
+package com.example.syncrowbackend.friendrequestroom.dto;
+
+import com.example.syncrowbackend.friendrequestroom.entity.FriendRequestPost;
+import com.fasterxml.jackson.annotation.JsonFormat;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import lombok.*;
+
+import java.time.LocalDateTime;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class FriendRequestPostDto {
+    private Long id;
+
+    @NotNull(message = "친구신청 작성한 사람의 아이디가 필요합니다.")
+    private Long user_id;
+
+    @NotBlank(message = "게시물에 제목은 필요합니다.")
+    private String title;
+
+    private String content;
+
+    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
+    private LocalDateTime createdAt = LocalDateTime.now();
+
+    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
+    private LocalDateTime modifiedAt = LocalDateTime.now();
+
+    @Builder
+    public static FriendRequestPostDto toDto(FriendRequestPost post) {
+        return FriendRequestPostDto.builder()
+                .id(post.getId())
+                .user_id(post.getUser().getId())
+                .title(post.getTitle())
+                .content(post.getContent())
+                .build();
```

Comment:
```suggestion
                .createdAt(post.getCreatedAt())
                .modifiedAt(post.getModifiedAt())
                .build();
```
dto 에 생성시간, 수정시간도 포함할까요? 🙂

Reply:
BaseTimeEntity에 @CreatedDate, @LastModifiedDate 어노테이션을 붙여 생성/수정 시간을 관리했고, LocalDateTime.now()값으로 Dto클래스의 필드에서 초기화를 진행했기 때문에 빌더패턴으로 관리한 생성자 부분엔 추가하지 않았습니다!

해당 어노테이션들은 Entity단에서 관리를 해주는 것 같아서  아래와 같이 코드 수정을 해볼까요? 
생성시간은 한번만 Entity단에서 create될때 생성하고, 수정시간은 toDto에서 실행될 때의 현재시간을 받아오게 바꿔봤습니다 😀
``` java
@JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
private LocalDateTime createdAt = LocalDateTime.now();

@JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
private LocalDateTime modifiedAt;

@Builder
public static FriendRequestPostDto toDto(FriendRequestPost post) {
    return FriendRequestPostDto.builder()
            .id(post.getId())
            .user_id(post.getUser().getId())
            .title(post.getTitle())
            .content(post.getContent())
            .modifiedAt(LocalDateTime.now())
            .build();
}
```

Reply:
네 그럼 수정 시간만 클라이언트에 전달해주게 되겠네요! 좋습니다 👍🏻